apluslms_roman/backends/docker.py

```diff
@@ -29,6 +29,9 @@ def _client(self):
         timeout = env.get('DOCKER_TIMEOUT', None)
         if timeout:
             kwargs['timeout'] = timeout
+        # host_path = env.get('DOCKER_HOST_PATH', None)
+        # if host_path:
+        #     print("Host source file path is", host_path)
         return docker.from_env(environment=env, **kwargs)
```

Comment:
`docker.from_env` is handy function, but it might become easier to use another one, when we change `env` to be a python dictionary with non-uppercase keys. Alternatively, this is the place to build minimalistic environment for the `from_env` from those variables we have available to use here.

ref: docker-py from_env: https://github.com/docker/docker-py/blob/master/docker/client.py#L85